examples/job/job.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Job
+metadata:
+  name: test-job
+spec:
+  parallelism: 2
+  completions: 5
+  selector:
```

Comment:
Selector should be set by default from template labels, so shouldn't need it here.